Q3DC/Q3DC.py

```diff
@@ -610,6 +732,20 @@ def findWidget(self, widget, objectName):
                     return resulting_widget
             return None
 
+    @staticmethod
+    def load_suggested_landmarks():
+        suggested_landmarks = defaultdict(list)
+        suggestions_path = \
+            Path(__file__).parent / 'Resources' / 'Data' / 'base_fiducial_legend.csv'
```

Comment:
to retrieve the path from the widget, use [self.resourcePath](https://github.com/Slicer/Slicer/blob/47deb76d7556e40de4e25e585c4b24a63a153da5/Base/Python/slicer/ScriptedLoadableModule.py#L95)

Reply:
Resolving, now tracked in https://github.com/DCBIA-OrthoLab/Q3DCExtension/issues/36